prep_managementplane_values.sh

```diff
@@ -29,6 +29,9 @@ $(awk '{printf "        %s\n", $0}' < xcp-central-cert.key)
     rootca: |
 $(awk '{printf "      %s\n", $0}' < ca.crt)
 spec:
+  certIssuer:
```

Comment:
if we do this should we get rid off... of 

https://github.com/tetrateio/tsb-helm-install-demos/blob/9017b41044270842a81b3c142a9521d49fab6058/prep_controlplane_values.sh#L33 and L36? what about elastic any chance there too?

Reply:
these don't get auto populated - we still need to provide CP the CA's.